### DISCUSSION TITLE
Support for GitHub releases?

### DISCUSSION THREAD

**Discussion by AndreZiviani** [argoproj/argo-cd#6603] [Q&A]
Some projects don't have helm charts and instead opted to distribute the manifests through GitHub releases (e.g. [kubernetes metrics-server](https://github.com/kubernetes-sigs/metrics-server))

Is this use case supported? If so, how can I do it?

**Comment by wtam2018** [comment]
You can download the manifest and push it to your repository.  Then, you can create an Argo CD application to deploy the manifest like this example https://github.com/argoproj/argocd-example-apps/tree/master/guestbook.

**Reply by AndreZiviani** [reply]
Sure but is there a way for argo to do it automatically? Track a specific version/release or latest?

**Reply by alexmt** [reply]
In some projects are leveraging kustomize to do it. It is possible to point kustomize to the file attached in release:

https://github.com/argoproj/argoproj-deployments/blob/6965cad7e9d81f37f491111bca8c6ce6e24e9da7/argocd-notifications/kustomization.yaml#L7

This approach has some caveats though: Argo CD won't notice a change in remote file for up to 24 hrs. User would have to trigger "hard refresh" to get changes immidiatelly.

**Reply by AndreZiviani** [reply]
Seem like a good solution, thanks

> This approach has some caveats though: Argo CD won't notice a change in remote file for up to 24 hrs.

Can you give more details on this?
